legal-api/src/legal_api/models/filing.py

```diff
@@ -95,6 +96,39 @@ class Source(Enum):
                 'CC': 'ALTER'
             }
         },
+        'amalgamation': {
+            'name': 'amalgamation',
```

Comment:
I think you need to have an entry like `'temporaryCorpTypeCode': 'TMP'`.  Take a look at the `incorporationApplication` and `registration` entries.

Maybe at the root or per amalgamation type.

Did you look into whether the temporary code("TMP") that IAs will work with auth and LEAR side?  I'm assuming it might but that was a question I added to the ticket indicating it required further investigation.

Reply:
so Kevin, as per the conversation above, please update this to include `temporaryCorpTypeCode` with a value of `ATMP`.  

You will also need to do a search in legal api to see where this is used and see if you need to make any updates to ensure a bootstrap business is created properly.

Reply:
Or try it from an amalg NR on My Business Register page, or from Namerequest UI.

Reply:
@kzdev420 can you confirm your latest code updates work locally when posting to the business endpoint to create a bootstrap amalgamation business?

Reply:
I am testing the Amalgamation locally.
But, I am getting an error. It creates the bootstrap but gets the error when registering the bootstrap.
Auth-API returns `500 Internal-Server-Error` ( Auth-API is not locally)
 
So, I am looking at the Auth-API